tests/offload_utils.hpp

```diff
@@ -0,0 +1,59 @@
+
+namespace {
```

Comment:
fine that this is anonymous unless we have multiple test translation units using this

Reply:
we do, but the point is that it needs to be local to each translation unit.